marlowe-playground-client/src/Help.purs

```diff
@@ -1,35 +1,35 @@
 module Help where
 
-import Prelude
 import Data.Array as Array
 import Data.String (fromCodePointArray, toCodePointArray)
-import Halogen.Classes (mAlignCenter, readMoreIconWhite, tAlignCenter)
-import Halogen.HTML (HTML, h4, hr, img, p_, text)
+import Halogen.Classes (readMoreIconWhite)
+import Halogen.HTML (ClassName(..), HTML, div, h4, hr, img, p_, text)
 import Halogen.HTML.Properties (alt, class_, src)
 import Marlowe.Holes (MarloweType(..))
+import Prelude (show, (<<<), (<>))
 import Types (HelpContext(..))
 
 toHTML :: forall p a. HelpContext -> Array (HTML p a)
-toHTML MarloweHelp =
-  [ img [ class_ mAlignCenter, src readMoreIconWhite, alt "read more icon" ]
-  , h4 [ class_ tAlignCenter ] [ text "Modelling contracts in Marlowe" ]
+toHTML helpType =
+  [ div [ class_ (ClassName "help-title") ]
+      [ img [ src readMoreIconWhite, alt "read more icon" ]
+      , h4 [] [ header helpType ]
+      ]
   , hr []
-  , p_ [ text "Marlowe is designed to support the execution of financial contracts on blockchain, and specifically to work on Cardano. Contracts are built by putting together a small number of constructs that in combination can be used to describe many different kinds of financial contract" ]
+  , p_ [ body helpType ]
   ]
+  where
+  header MarloweHelp = text "Modelling contracts in Marlowe"
```

Comment:
Doesn't really matter, but `header` and `body` are slightly tricky choices given the're HTML tag names. (And common ones at that.)

Reply:
changed to `headerText` and `bodyText`